src/com/massivecraft/massivetickets/cmd/CmdTicketsClose.java

```diff
@@ -0,0 +1,69 @@
+package com.massivecraft.massivetickets.cmd;
+
+import com.massivecraft.massivecore.MassiveException;
+import com.massivecraft.massivecore.command.requirement.RequirementHasPerm;
+import com.massivecraft.massivetickets.MassiveTickets;
+import com.massivecraft.massivetickets.Perm;
+import com.massivecraft.massivetickets.cmd.type.TypeMPlayer;
+import com.massivecraft.massivetickets.entity.MConf;
+import com.massivecraft.massivetickets.entity.MPlayer;
+
+import java.util.List;
+
+public class CmdTicketsClose extends MassiveTicketsCommand
+{
+	// -------------------------------------------- //
+	// INSTANCE
+	// -------------------------------------------- //
+	
+	private static CmdTicketsClose i = new CmdTicketsClose() { @Override public List<String> getAliases() { return MConf.get().aliasesOuterTicketsClose; } };
+	public static CmdTicketsClose get() { return i; }
+	
+	// -------------------------------------------- //
+	// CONSTRUCT
+	// -------------------------------------------- //
+	
+	public CmdTicketsClose()
+	{
+		// Parameters
+		this.addParameter(TypeMPlayer.getAny(), "player");
+		
+		// Requirements
+		this.addRequirements(RequirementHasPerm.get(Perm.CLOSE));
+	}
+	
+	// -------------------------------------------- //
+	// OVERRIDE
+	// -------------------------------------------- //
+	
+	@Override
+	public List<String> getAliases()
+	{
+		return MConf.get().aliasesInnerTicketsClose;
+	}
+	
+	@Override
+	public void perform() throws MassiveException
+	{
+		// Args
+		MPlayer ticket = this.readArg();
+		
+		// Force Sync
+		ticket.sync();
+		
+		// Is there a ticket?
+		if (!ticket.hasMessage())
+		{
+			msg("<white>%s <b>has not created a ticket.", ticket.getDisplayName(null));
```

Comment:
You are supplying `null` but the correct way would be `ticket.getDisplayName(sender)`. We do have the `sender` and want to colorise accordingly.

Reply:
Interstingly, this line was just a copy (as was most of the command, with slight modifications) of the Done command. Looking at the usages, many occurences of getDisplayName in the project use null instead of sender or the correct watcher object.
I might go through and revise these in a separate PR.